__tests__/app.test.js

```diff
@@ -29,4 +31,21 @@ describe(('GET/api/topics'), () => {
               ])
         })
     })
-})
\ No newline at end of file
+})
+describe("GET/api", () => {
+  test('200: responds with a full list of created endpoints as a JSON object', () => {
+    const expectedEndpoints = fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
+    
+    const actualEndpoints = request(app)
+    .get('/api')
+    .expect(200)
+    .then(({body}) => {
+      return body
+    })
+
+    Promise.all([expectedEndpoints, actualEndpoints])
```

Comment:
This issue might be with the Promise.all not being returned, or not being in the .then chain. But requiring the data in from the file will lessen the need for this.